### ISSUE TITLE
Localization Alignment Issue

### ISSUE THREAD

**Issue by bitigchi** [FDOS/fdisk#53]
In the section https://github.com/FDOS/fdisk/blob/429e763a36da730d3929eef00707759a1aaebded/source/fdisk/nls/en_utf8.txt#L222, there seems to be some invisible characters to fix the layout. Is it possible to get a more precise commentary about how the alignment should work for every entry?

For instance, if I modify the first entry as such:

```
9.10:\n\nPartition   Status   Mbytes   Description      Usage    Start CHS       End CHS\n
9.10:\n\nBölüntü     Status   Mbytes   Description      Usage    Start CHS       End CHS\n
```

the Status starts at a different line number than the original (22 vs. 25).

How to properly approach these strings?

**Comment by boeckmann** [comment]
The alignment is done via space character. I can add comments if the fields are left or right aligned. Do you mean that? The string 9.10. is the table header of the command line argument /info. like shown in the appended picture for my german translation.
<img width="718" alt="Bildschirmfoto 2023-07-07 um 21 54 59" src="https://github.com/FDOS/fdisk/assets/6333969/4e003618-f810-4ade-bfc0-0fccff10eb86">

**Comment by boeckmann** [comment]
If you are unsure I would recommend to to a "raw" translation of the headers. I will have a look at them and will come back at you if some strings do not fit.

**Comment by bitigchi** [comment]
I think I understand why my character counter in Vim was jumping, it should be related to the encoding and the special characters.

Nonetheless, to be 100% sure, how does the below use look?

```
9.10:\n\nPartition   Status   Mbytes   Description      Usage    Start CHS       End CHS\n
9.10:\n\nBölüntü     Durum    Mbayt    Açıklama      Kullanım      CHS Baş       CHS Son\n
9.11:Drv Volume Label  Mbytes  System   Usage    Start CHS       End CHS\n
9.11:Src Bölüm  Etiket Mbayt   Sistem   Kulln      CHS Baş       CHS Son\n
9.12:Disk   Drv   Mbytes    Free  Usage
9.12:Disk   Src   Mbayt     Boş   Kullanım
```

I'd appreciate another couple of eyes looking at it. If possible I'd like to use the full version of `Kulln` on the second string as `Kullanım`, but this one is okay as well.

**Comment by boeckmann** [comment]
Thank you very much for contributing this translation! This is turkish language?

You can perfectly use the long version at the second string. You can align it centered to the percent values below it (see screenshot below "Nutzung"). Simply make sure that CHS Baş stays right-aligned with Start CHS the line above in your code snipped posted above.

Like so:
```
9.11:Drv Volume Label  Mbytes  System   Usage    Start CHS       End CHS\n
9.11:Src Bölüm  Etiket Mbayt   Sistem  Kullanım    CHS Baş       CHS Son\n
```

<img width="614" alt="Bildschirmfoto 2023-07-07 um 22 17 50" src="https://github.com/FDOS/fdisk/assets/6333969/799d8749-de98-44a0-9cd9-c72b48d5504a">

**Comment by bitigchi** [comment]
Yes, Turkish indeed!

Thank your very much for your help! I'll have the translations ready probably tomorrow!

**Comment by boeckmann** [comment]
Small tweak: would right-align Mbayt with Mbytes, and Volume Label is one field, so I would use only one space in `Bölüm Etiket`:
```
9.11:Drv Volume Label  Mbytes  System   Usage    Start CHS       End CHS\n
9.11:Src Bölüm Etiket   Mbayt  Sistem  Kullanım    CHS Baş       CHS Son\n
```

**Comment by boeckmann** [comment]
> Yes, Turkish indeed!
> 
> Thank your very much for your help! I'll have the translations ready probably tomorrow!

Don't hurry. Do you know how to test the translation inside FDISK?

**Comment by bitigchi** [comment]
Not really. I was thinking of compiling it somehow, and trying it under DOSBox or something. Is there an easier way?

**Comment by boeckmann** [comment]
If you want to, you may use this ready-made package for translator, which includes everything that is needed. Simply download all files and extract the ZIP archive. Follow the howto. If something is unclear, do not hesitate to ask me, that I can improve it.

Alternatively, you may simply send me the translated file when you are finished. Then I will check it to see if everything fits. Like you want.

https://nextcloud.iww.rwth-aachen.de/index.php/s/w8nmaH8gMB9H3Rj

**Comment by bitigchi** [comment]
I will have a look at this tomorrow, due to the late hour, thanks in advance!

I will definitely ask, if there is something I don't understand.

**Comment by bitigchi** [comment]
Eh, it looks like @cardpuncher beat me to it. :)

**Comment by cardpuncher** [comment]
On 8.07.2023 21:01, Emir SARI wrote:
> Eh, it looks like @cardpuncher <https://github.com/cardpuncher> beat me 
> to it. :)

Hey Emir,

Nice to hear from you :) I took a look at open pull requests and issue 
reports just in case, but not to the closed ones. Anyway, right now I'm 
struggling to test the translations because FDISK is only displayed in 
English on my side, until I get that working nicely feel free to check 
my translation and fix my mistakes and improve the sentences!

**Comment by boeckmann** [comment]
I created a new issue for the Turkish translation. Thank you guys for taking care of it! And sorry that I did not manage to prevent the double work resulting from you working on it in parallel.
